requirements.txt

```
@@ -1,11 +1,11 @@
 annotated-types==0.7.0
 anyio==4.9.0
-boto3==1.35.99
-botocore==1.35.99
+boto3==1.38.36
+botocore==1.38.36
```

The End-to-End Tests job is failing because of the issues recorded [here](http://github.com/ral-facilities/object-storage-api/actions/runs/15686747493/job/44191807370?pr=175#step:6:18410).
```suggestion
boto3==1.35.99
botocore==1.35.99
```